packages/flame_tiled/lib/src/renderable_tile_map.dart

```diff
@@ -41,6 +41,45 @@ class RenderableTiledMap {
       ? batchesByLayer.first
       : <String, SpriteBatch>{};
 
+  /// Changes the visibility of the corresponding layer, if different
+  void setLayerVisibility(int layerId, bool visibility) {
+    if (map.layers[layerId].visible != visibility) {
+      map.layers[layerId].visible = visibility;
+      refreshCache();
+    }
+  }
+
+  /// Gets the visibility of the corresponding layer
+  bool getLayerVisibility(int layerId) {
+    return map.layers[layerId].visible;
+  }
+
+  /// Changes the Gid of the corresponding layer at the given position, if different
+  void setTileData({
+    required int layerId,
+    required int x,
+    required int y,
+    required Gid gid,
+  }) {
+    final layer = map.layers[layerId] as TileLayer;
+    final td = layer.tileData;
+    if (td != null) {
+      td[y][x] = gid;
+      refreshCache();
+    }
+  }
+
+  /// Gets the Gid  of the corresponding layer at the given position
+  Gid? getTileData({required int layerId, required int x, required int y}) {
+    final layer = map.layers[layerId] as TileLayer;
+    final td = layer.tileData;
+    if (td != null) {
+      return td[y][x];
+    } else {
+      return null;
+    }
```

Comment:
```suggestion
    return td?[y][x];
```
Should be possible to write like that I think?

Reply:
I think you misunderstood what I meant, sent a new suggested change. :)

Reply:
Ah, OK, next time I simply click "Commit suggestion"! :-))